pkg/kubelet/oom/oom_watcher_linux_test.go

```diff
@@ -52,6 +66,9 @@ func TestStartingWatcher(t *testing.T) {
 // TestWatcherRecordsEventsForOomEvents ensures that our OomInstances coming
 // from `StreamOoms` are translated into events in our recorder.
 func TestWatcherRecordsEventsForOomEvents(t *testing.T) {
+	if skipTest {
```

Comment:
I don't think you need to skip this test case or any of the following ones, since they use a fake streamer (rather than calling the real `NewWatcher` function).